build.sc

```diff
@@ -140,15 +151,11 @@ object pipelines extends CommonModule {
   )
 
   /** Module dependencies. */
-  override def moduleDeps: Seq[commons.type] = Seq(commons)
+  override def moduleDeps      = Seq(commons, commons.scoverage)
```

Comment:
This isn't ideal because `PublishModule` requires this upper type bound to be `PublishModule` and not `ScalaModule`.